app/javascript/components/enrollment/steps/AdditionalPlannedTravel.js

```diff
@@ -20,6 +20,14 @@ class AdditionalPlannedTravel extends React.Component {
     let value = event.target.type === 'checkbox' ? event.target.checked : event.target.value;
     let current = this.state.current;
     let modified = this.state.modified;
+
+    const trimFields = ['additional_planned_travel_destination', 'additional_planned_travel_port_of_departure'];
+    if (event?.target?.id && trimFields.includes(event.target.id)) {
+      if (event.target.value.trim() == '') {
```

Comment:
A slight nitpick, but in general in JS we like to use three equals-signs (`===`) over two (`==`).

Reply:
Thanks for the suggestion! Will push an commit with updated shortly.